.env.tmp

```diff
@@ -5,7 +5,9 @@ API_KEY='your-api-key'
 AUTH0_AUDIENCE='https://otp-middleware'
 AUTH0_DOMAIN='you.auth0.com'
 AUTH0_CLIENT_ID='your-client-id'
+OTP_DEVGROUP_URL='https://groups.google.com/g/opentripplanner-dev'
 OTP_UI_URL='https://otp-ui'
+SITE_LOGO='https://path/to/logo.png'
```

Comment:
What does this do? Is there a way to add comments to this file?

Reply:
Ping @binh-dam-ibigroup

Reply:
Hmm for some reason I thought I answered already. `SITE_LOGO` was introduced by https://github.com/ibi-group/otp-admin-ui/commit/24045651bcde1c8b7399430dfbe1b6fe31441a17, and reviewing variables in the config template file made me add it as it was missing.